docs/whatsnew/4.0.rst

```diff
@@ -28,6 +28,15 @@ Improved return types of HEK queries
 The 'event_endtime', 'event_starttime' and 'event_peaktime' columns in a HEK
 query are now returned as `~astropy.time.Time` objects.
 
+
+Drawing Carrington coordinate grids
+===================================
+It is now easy to draw Carrington coordinate grids on top of maps using
+:meth:`sunpy.map.GenericMap.draw_grid` by specifying ``system='carrington'``.
+
+.. minigallery:: sunpy.map.GenericMap.draw_grid
```

Comment:
Ugh, the `minigallery` call pulls in way too many examples, with only 1 of the 15 actually showing the new functionality.  I say change this to a Sphinx link to just this PR's example

Reply:
Hopefully I spelt interlink correctly.